docs/UserGuide.md

```diff
@@ -48,9 +48,9 @@ This user guide provides in-depth descriptions of all the available features in
 1. Ensure you have Java `11` or above installed in your Computer.
 
 1. Download the latest `guestbook.jar` from [here](https://github.com/AY2223S1-CS2103T-W16-1/tp/releases).
-
+   ![Ui](images/download.png)
```

Comment:
For the image, since our 'here' already takes us to the releases page, should we just have a zoom in picture of where to download? For e.g like this
![Screenshot 2022-10-26 at 3 21 29 PM](https://user-images.githubusercontent.com/48787208/197960591-b2dc9fc0-52cb-42b7-8244-39982d41c488.png)

Also, should we make the image more informative by directing the user to click the guestbook.jar to download?